Source/Core/DolphinLibretro/Input.cpp

```diff
@@ -541,40 +569,40 @@ void retro_set_controller_port_device(unsigned port, unsigned device)
-      ccButtons->SetControlExpression(4, "L2");                             // ZL
-      ccButtons->SetControlExpression(5, "R2");                             // ZR
-      ccButtons->SetControlExpression(6, "Select");                         // -
-      ccButtons->SetControlExpression(7, "Start");                          // +
-      ccButtons->SetControlExpression(8, "R3");                             // Home
-      ccTriggers->SetControlExpression(0, "L");                             // L-trigger
-      ccTriggers->SetControlExpression(1, "R");                             // R-trigger
-      ccTriggers->SetControlExpression(2, "`" + devAnalog + "Trigger0+`");  // L-trigger Analog
-      ccTriggers->SetControlExpression(3, "`" + devAnalog + "Trigger1+`");  // R-trigger Analog
-      ccDpad->SetControlExpression(0, "Up");                                // Up
-      ccDpad->SetControlExpression(1, "Down");                              // Down
-      ccDpad->SetControlExpression(2, "Left");                              // Left
-      ccDpad->SetControlExpression(3, "Right");                             // Right
-      ccLeftStick->SetControlExpression(0, "`" + devAnalog + ":Y0-`");      // Up
-      ccLeftStick->SetControlExpression(1, "`" + devAnalog + ":Y0+`");      // Down
-      ccLeftStick->SetControlExpression(2, "`" + devAnalog + ":X0-`");      // Left
-      ccLeftStick->SetControlExpression(3, "`" + devAnalog + ":X0+`");      // Right
-      ccRightStick->SetControlExpression(0, "`" + devAnalog + ":Y1-`");     // Up
-      ccRightStick->SetControlExpression(1, "`" + devAnalog + ":Y1+`");     // Down
-      ccRightStick->SetControlExpression(2, "`" + devAnalog + ":X1-`");     // Left
-      ccRightStick->SetControlExpression(3, "`" + devAnalog + ":X1+`");     // Right
+      ccButtons->SetControlExpression(0, "A");                               // A
+      ccButtons->SetControlExpression(1, "B");                               // B
+      ccButtons->SetControlExpression(2, "X");                               // X
+      ccButtons->SetControlExpression(3, "Y");                               // Y
+      ccButtons->SetControlExpression(4, "L");                               // ZL
+      ccButtons->SetControlExpression(5, "R");                               // ZR
+      ccButtons->SetControlExpression(6, "Select");                          // -
+      ccButtons->SetControlExpression(7, "Start");                           // +
+      ccButtons->SetControlExpression(8, "R3");                              // Home
+      ccTriggers->SetControlExpression(0, "`" + devAnalog + ":Trigger0+`");  // L-trigger
+      ccTriggers->SetControlExpression(1, "`" + devAnalog + ":Trigger1+`");  // R-trigger
+      ccTriggers->SetControlExpression(2, "`" + devAnalog + ":Trigger0+`");  // L-trigger Analog
+      ccTriggers->SetControlExpression(3, "`" + devAnalog + ":Trigger1+`");  // R-trigger Analog
```

Comment:
same as above.

Reply:
Do you want me to use the same logic for GC triggers here? I didn't actually change the logic here.

Reply:
it used to be just `L` and `R` for '0' and '1'. but since the digital input would override the analog one, it does make sense to use only analog here.
you can keep it this way then, unless the changes I suggested above to `MixedTriggers.cpp` would work better.